### PR TITLE
adding SKR troubleshooting guide

### PR DESCRIPTION
**Pull request by hgarvison**

### CODE REVIEW

**Review by SethHollandsworth — APPROVED** [review]
good start 👍

File: examples/skr/TROUBLESHOOTING.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+## 401 Unauthorized Error
+
+When running the importkey tool, you may see the following error:
+
+```
+pulling AKV response body failed: http response status equal to 401 Unauthorized
+```
+
+To resolve this issue, generate a new bearer token and copy it into the importkeyconfig.json.
```
Comment (BryceDFisher) [inline examples/skr/TROUBLESHOOTING.md]: We can skip the boilerplate "To resolve this issue"...

File: examples/skr/TROUBLESHOOTING.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+When checking the log output of the SKR container, you may see the following error:
+
+```
+err: pulling AKV response body failed: http response status equal to 404 Not Found
+```
+
+To resolve this issue, ensure that:
+
+- the key name ("kid" field) in the importkeyconfig.json file matches the key name ("SkrClientKID") field in the ARM template
```
Comment (BryceDFisher) [inline examples/skr/TROUBLESHOOTING.md]: And matches the kid in the importkeyconfig.json?

File: examples/skr/TROUBLESHOOTING.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+## HTTP GET Failed Error
+
+When checking the log output of the SKR container, you may see the following error:
```
Comment (BryceDFisher) [inline examples/skr/TROUBLESHOOTING.md:47]: We can skip the boilerplate "When checking the log output of the SKR container".

File: examples/skr/README.md [hunk 4]
```diff
@@ -131,27 +106,27 @@ A fake encryption key is used in the command below to see the key get released.
...
 
-Upon successful import completion, you should see something similar to the following: 
+Upon successful import completion, you should see something similar to the following:
 
 ```
 [34 71 33 117 113 25 191 84 199 236 137 166 201 103 83 20 203 233 66 236 121 110 223 2 122 99 106 20 22 212 49 224]
-https://accmhsm.managedhsm.azure.net/keys/doc-sample-key-release/8659****0cdff08
-{"version":"1.0.0","anyOf":[{"authority":"https://sharedeus2.eus2.test.attest.azure.net","allOf":[{"claim":"x-ms-sevsnpvm-hostdata","equals":"aaa7***7cc09d"},{"claim":"x-ms-compliance-status","equals":"azure-compliant-uvm"},{"claim":"x-ms-sevsnpvm-is-debuggable","equals":"false"}]}]}
+https://<mhsm-name>.managedhsm.azure.net/keys/doc-sample-key-release/8659****0cdff08
+{"version":"1.0.0","anyOf":[{"authority":"<authority-url-name>","allOf":[{"claim":"x-ms-sevsnpvm-hostdata","equals":"aaa7***7cc09d"},{"claim":"x-ms-compliance-status","equals":"azure-compliant-uvm"},{"claim":"x-ms-sevsnpvm-is-debuggable","equals":"false"}]}]}
```
Comment (BryceDFisher) [inline examples/skr/README.md:114]: I think this authority is pretty stable for people in eastus.  We should at least provide a link to the doc on where to find these.

File: examples/skr/TROUBLESHOOTING.md (hunk 3 above)
Comment (hgarvison) [reply examples/skr/TROUBLESHOOTING.md:47]: In one case the error is not seen in the log output of the SKR container though, so I think indicating where the error is found is useful.

File: examples/skr/README.md (hunk 4 above)
Comment (hgarvison) [reply examples/skr/README.md:114]: Info on the authority is found in the readme when telling people how to create an MAA instance. We just didn't want our test MAA/AKV names in the documentation

**Pull request merged**